pkg/upgrade/upgrade.go

```diff
@@ -254,6 +266,33 @@ func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS
 	return nil
 }
 
+func jobGetDashboardWatsonResources(ns string) []JobSpec {
+	pathName := []string{"metadata", "name"}
```

Comment:
or maybe not use variable but directly set it ?

Reply:
> how you feel name it as

Fine. Naming is a BIG question here, any suggestion is welcome!

Reply:
> or maybe not use variable but directly set it ?

it's repeated several times (except "metadata", "name" here, leftover from the previous version), I would prefer to keep them in variables since it's shorter. But no problem to change is you think it's better.

Reply:
ok, then lets get the change for the variable names instead of hardcode path

Reply:
> ok, then lets get the change for the variable names instead of hardcode path
Updated. I just kept "metadata" lowercase as the field name.